packages/block-library/src/block/edit.js

```diff
@@ -38,7 +38,8 @@ import { name as patternBlockName } from './index';
 import { unlock } from '../lock-unlock';
 
 const { useLayoutClasses } = unlock( blockEditorPrivateApis );
-const { isOverridableBlock } = unlock( patternsPrivateApis );
+const { isOverridableBlock, PARTIAL_SYNCING_SUPPORTED_BLOCKS } =
+	unlock( patternsPrivateApis );
```

Comment:
Optional: `getOverridableAttributes` could be moved to the patterns package too. I quite like the idea of these 'source of truth' functions living there.